site/profile/manifests/cvmfs.pp

```diff
@@ -48,6 +49,15 @@
     require => Package['cvmfs']
   }
 
+  file { 'etc/cvfms/config.d':
```

Comment:
This path is missing is front slash `/etc/cvfms/config.d'`.

Reply:
Add the `/` and change the name of the file